Update config-settings.rst (#3787)
mattermost/docs: source/administration/config-settings.rst

File Storage
~~~~~~~~~~~~~~~~~~~~~~~~~

> Mattermost currently supports storing files on the local filesystem and Amazon S3 or object storage systems which implement the S3 API. We officially support AWS S3 and Minio.

Reviewer: ```suggestion
Mattermost currently supports storing files on the local filesystem and Amazon S3 or object storage systems which implement the S3 API. We officially support AWS S3 and MinIO.
```

.. note::
  

>   Other S3 compatible systems may work, but are not officially supported. We have tested Mattermost with `Digital Ocean Spaces <https://www.digitalocean.com/docs/spaces/>`_ products but not all S3 compatible containers on the market. If you are looking to use other S3 compatible containers we advise completing your own testing.

Reviewer: ```suggestion
We have tested Mattermost with `Digital Ocean Spaces <https://www.digitalocean.com/docs/spaces/>`_ products but not all S3-compatible containers on the market. Other S3-compatible systems may work, but are not officially supported so we advise completing your own testing prior to use.
```
